README.md

```diff
@@ -75,7 +75,7 @@ in which case you may not even need the asset pipeline. This is mostly relevant
 * Ruby 2.2+
 * Rails 4.2+
 * Node.js 6.4.0+
-* Yarn 0.20.1+
+* Yarn 0.20.1+ or NPM 4.2.0+
```

Comment:
Minor nit, but "npm" is always lowercase: https://twitter.com/seldo/status/850042251635867648